template.yaml

```diff
@@ -90,6 +90,8 @@ Conditions:
   EnableCloudFront: !Or
     - !Equals [ !Ref Environment, dev ]
     - !Equals [ !Ref Environment, build ]
+    - !Equals [ !Ref Environment, staging ]
```

Comment:
I dont see CloudFrontWAFv2ACLAssociation added in here like in another PR. Are we saying that it already has this?

Reply:
Already merged on F2F -> https://github.com/govuk-one-login/ipv-cri-f2f-front/blob/main/template.yaml#L714